grakn-bootup/src/main/java/ai/grakn/bootup/ProcessHandler.java

```diff
@@ -19,20 +19,29 @@
 package ai.grakn.bootup;
 
 /**
+ * The methods of this interface are used to define a common expectation of what processes should offer.
+ * The actual {@link ProcessHandler} interface is never directly accessed.
+ * Therefore the need for the suppressions below
```

Comment:
I'd rather remove unused interfaces like this - but I think this is a personal opinion so keeping it is all good.

Reply:
Killing it.